layouts/shortcodes/pac-preview.html

```diff
@@ -0,0 +1,8 @@
+<div class="note note-info">
+    <p>
+        <i class="fas fa-info-circle pr-2"></i>
+        Policy as Code is a beta feature and is subject to breaking changes.
+        This feature is currently being previewed to get access, submit a request <a
```

Comment:
Does someone need to contact us if they just want to use the `--policy-pack` style of local execution?

Reply:
Nope -- I'll make this clearer.